src/bcs/serializable/moveStructs.ts

```diff
@@ -432,6 +447,32 @@ export class MoveOption<T extends Serializable & EntryFunctionArgument>
     return new MoveOption<MoveString>(value !== null && value !== undefined ? new MoveString(value) : undefined);
   }
 
+  /**
+   * Factory method to generate a MoveOption<AccountAddress> from `HexInput`, `AccountAddress` or `undefined`
+   *
+   * @example
+   * MoveOption.AccountAddress("0x1").isSome() === true;
+   * MoveOption.AccountAddress("0xbeefcafe").isSome() === true;
+   * MoveOption.AccountAddress().isSome() === false;
+   * MoveOption.AccountAddress(undefined).isSome() === false;
+   * @params value: the value used to fill the MoveOption. If `value` is undefined
+   * the resulting MoveOption's .isSome() method will return false.
+   * @returns a MoveOption<AccountAddress> with an inner value `value`
+   */
+  static AccountAddress(value?: HexInput | AccountAddress | null): MoveOption<AccountAddress> {
+    let accountAddress: AccountAddress | undefined;
+    if (value !== null && value !== undefined) {
+      if (value instanceof AccountAddress) {
+        accountAddress = value;
+      } else {
+        accountAddress = AccountAddress.fromHexInputRelaxed(value);
+      }
+    } else {
```

Comment:
We should merge this in with this change https://github.com/aptos-labs/aptos-ts-sdk/pull/145 but it keeps bloating more and more for me every time I'm fixing one thing, so I'm trying to get it working soon.